docs/flink-connector/flink-catalog-iceberg.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Flink connector Iceberg catalog"
+slug: /flink-connector/flink-catalog-iceberg
+keyword: flink connector iceberg catalog
+license: "This software is licensed under the Apache License version 2."
+---
+
+The Apache Gravitino Flink connector can be used to read and write Iceberg tables, with the metadata managed by the Gravitino server.
+To enable the Flink connector, you must download the Iceberg Flink runtime JAR and place it in the Flink classpath.
+
+## Capabilities
+
+#### Supported DML and DDL operations:
+
+- `CREATE CATALOG`
+- `CREATE DATABASE`
+- `CREATE TABLE`
+- `DROP TABLE`
+- `ALTER TABLE`
+- `INSERT INTO & OVERWRITE`
+- `SELECT`
+
+#### Operations not supported:
+
+- Partition operations
+- View operations
+- Metadata tables, like:
+  - `{iceberg_catalog}.{iceberg_database}.{iceberg_table}&snapshots`
+- Querying UDF
+  - `UPDATE` clause
```

Comment:
UPDATE & DELETE clause are not under `Query UDF`?

Reply:
OOH, what I means is the markdown style,     UPDATE & DELETE clause seems should be the same level with `Query UDF`, but now there are extra spaces.

Reply:
ok, sorry

Reply:
could you place `UPDATE & DELETE` and `Query UDF` in same level?

Reply:
@sunxiaojian WDYT?

Reply:
> @sunxiaojian WDYT?

fixed